data_processing/common/constants.py

```diff
@@ -0,0 +1,19 @@
+'''
+Created on November 16, 2020
+
+@author: rosed2@mskcc.org
+'''
+import os
+# Application Constants
+
+#### table names ####
+# clinical
+DIAGNOSIS_TABLE		='tables/diagnosis'
+MEDICATION_TABLE	='tables/medication'
+PATIENT_TABLE		='tables/patient'
+
+# radiology
+DICOM_TABLE			='tables/dicom'
+SCAN_TABLE 			='tables/scan'
+SCAN_ANNOTATION_TABLE		='tables/scan_annotation'
```

Comment:
Tiny bit of DRY here. Can help to avoid typos in the future - 

`
BASE='tables'

DIAGNOSIS_TABLE		=BASE+'/diagnosis'

MEDICATION_TABLE	=BASE+'/medication'
`

Reply:
Agreed. I made the change. Thanks for reviewing!